101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mediaServiceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services account. A Media Services account name is globally unique, all lowercase numbers or letters with no spaces."
+      }
+    }
+  },
+  "variables": {
+    "storageName": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "storageType": "Standard_LRS"
+  },
+  "resources": [
+    {
+      "name": "[parameters('mediaServiceName')]",
+      "type": "Microsoft.Media/mediaServices",
+      "apiVersion": "2015-10-01",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[concat('microsoft.storage/storageaccounts/', variables('storageName'))]"
```

Comment:
Why not, I'm not seeing the reference of this best practice here https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-define-dependencies#dependson. BTW, the update is done. Thanks.